x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -117,29 +128,24 @@ export function WaterfallItem({
 
   const width = (item.duration / totalDuration) * 100;
   const left = ((item.offset + item.skew) / totalDuration) * 100;
-  const Label = item.docType === 'transaction' ? TransactionLabel : SpanLabel;
+  const Label = item.docType === 'span' ? SpanLabel : TransactionLabel;
```

Comment:
lol! I changed it to something else entirely in between, then reconsidered, and thought I had put it back the way it was but I guess I chose "span" instead of "transaction" for the ternary condition. Cool diff!